demo/ios/OrcaDemo/OrcaDemo/ViewModel.swift

```diff
@@ -129,23 +337,25 @@ class ViewModel: ObservableObject {
     public func isValid(text: String) {
         do {
             let characters = try orca.validCharacters
-            let regex = try NSRegularExpression(
-                pattern: "[^\(characters.joined(separator: ""))\\s{}|']",
-                options: .caseInsensitive)
-            let range = NSRange(text.startIndex..<text.endIndex, in: text)
-            let matches = regex.matches(in: text, range: range)
-
-            let unexpectedCharacters = NSOrderedSet(array: matches.map {
-                String(text[Range($0.range, in: text)!])
-            })
+            
+            var nonAllowedCharacters = [Character]()
```

Comment:
a good example of setting init-time contants during init leading to more code - here in `isValied`, the only reason it is wrapped in a try/catch is because `try orca.validCharacters` is making a call into the lib and trying to convert the response. If we do this during init, they could just simply call `orca.validChacters` and it would return a swift value that's attached to the orca object